frontend/src/app/patients/__snapshots__/EditPatient.test.tsx.snap

```diff
@@ -273,6 +271,1649 @@ Object {
                       >
                         Preferred language
                       </label>
+                      <div
```

Comment:
Not that I have a reason to think it matters, but did we ever figure out why this big snapshot diff happened?

Reply:
yes, its because I removed this mock https://github.com/CDCgov/prime-simplereport/pull/6584/files#diff-18ca4e8fbb71d8294d96670dc4f11fb51569718748aad3b0717f4697ed3df147L22